assemble/bin/accumulo-cluster

```diff
@@ -119,6 +119,24 @@ function control_service() {
   host="$2"
   service="$3"
 
+  #
+  # The server processes take arguments (e.g. -a, -p, -o, -q [in the case of the Compactor]).
+  #
+  if [[ $# -gt 3 ]]; then
+    EXTRA_ARGS="${@:4}"
+  else
+    EXTRA_ARGS=""
+  fi
+
+  #
+  # If the host is an IP address, then pass the IP address in the -a argument to the server. If
+  # this is not done, then the default IP address if 0.0.0.0 is assigned and the host name is used
+  # in the ZooKeeper address (which may not be resolvable in some systems)
+  #
+  if [[ $host =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    EXTRA_ARGS="-a $host $EXTRA_ARGS"
```

Comment:
Instead of reusing the `EXTRA_ARGS` mechanism, it would probably be better to just add a new variable for this. At the very least, it would make this diff more intuitive and easier to review.